understanding/20/error-identification.html

```diff
@@ -191,6 +188,12 @@ <h4>Situation A: If a form contains fields for which information from the user i
                   <a href="https://www.w3.org/WAI/WCAG21/Techniques/general/G83" class="general">Providing a text description that identifies the field as mandatory</a>
                   									         
                </li>
+
+               <li>
+                  										           
+                  <a href="https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA2" class="aria">ARIA2: Identifying required fields with the "required" property</a>
+                  									         
+               </li>
```

Comment:
* In subsection "Sufficient Techniques for Error Identification" there are several empty list items / links that should probably be removed.
* There are two empty sections, "Resources for Error Identification" and, under "Techniques for Error Identification", "Failures for Error Identification". I guess these should be removed?

Reply:
those were parts of the file i hadn't touched, and looking at the live version, it seems the xslt/publish script silently removes there. happy to rip them out of the file itself ...

Reply:
Agree with patrick, I think this is actually just a case where the preview rendering is inaccurate. This happens because some of the links here are written in the source like this:

```html
               <li>

                  <a href="https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA21" class="aria"></a>
                  
               </li>
```

...which looks like empty list items as rendered by the preview, but when processed to build the live version, actually results in the `<a>` elements getting filled in using the title of the page (you can see an "ARIA21" link like the above example correctly in the rendered live page)